core/store/migrate/migrations/0188_create_automation_upkeep_state.sql

```diff
@@ -0,0 +1,18 @@
+-- +goose Up
+
+CREATE TABLE evm_upkeep_state (
```

Comment:
add fields: 
- upkeep_id (`BIGSERIAL`)
- ineligibility_reason (`NUMERIC`)

Reply:
Why `BIGSERIAL` on upkeep_id? This isn't an auto-incrementing value is it? Where do you want to get ineligibility reason from or should I allow null on that value?

Reply:
NMV on the ineligibility reason. That comes from the check result